src/pages/projects/the-beginner-s-guide-to-vue-3/index.tsx

```diff
@@ -0,0 +1,230 @@
+    label: 'App Design',
+  })
+  const Submission: any = find(course?.projects?.content, {label: 'Submission'})
+
+  const {productCard, productPage} = course?.projects
+
+  console.log(course)
+
+  return (
+    <>
+      <NextSeo
+        description={course.description}
+        title={title}
+        titleTemplate={'%s | egghead.io'}
+        twitter={{
+          site: `@eggheadio`,
+          cardType: 'summary_large_image',
+        }}
+        openGraph={{
+          title,
+          description: course.description,
+          site_name: 'egghead',
+          images: [
+            {
+              url: ogImage,
+            },
+          ],
+        }}
+      />
+      <div className="bg-gray-50 dark:bg-gray-900 sm:-my-5 -my-3 -mx-5 p-5">
+        <div className="mb-10 pb-10 xl:px-0 px-5 max-w-screen-xl mx-auto">
+          <div className="mt-10 mb-16 text-center">
+            <div className="mb-10">
+              <Image priority src={image} height="270" width="270" />
+            </div>
+            <p className="text-lg md:text-2xl leading-6 text-gray-500">
+              Portfolio Project
+            </p>
+            <h1 className="text-2xl md:text-4xl font-bold mt-2">
+              {course.projects.title}
+            </h1>
+          </div>
+          <div className="flex flex-col justify-center items-start mx-auto max-w-screen-md mb-16 w-full px-4">
+            <main className="prose dark:prose-dark prose-lg max-w-none w-full">
+              <Markdown>{introduction.text}</Markdown>
+              <Markdown>{projectBrief.text}</Markdown>
+              <section className="grid grid-cols-2 gap-4">
+                <Markdown className="col-span-1">
+                  {appRequirements.text}
+                </Markdown>
+                <Markdown className="rounded mt-20">{appData.text}</Markdown>
+              </section>
+              <Markdown>{developmentStandards.text}</Markdown>
+
+              <ProjectCTA
+                title="Show you're hard won Vue skills. Make it stick."
+                meta="Fork the codesandbox and get started with your solution."
+                buttonText="Start the Vue Fundamentals Project"
+                projectLink={course.projects.projectLink}
+              />
+              <Markdown>{stretchGoal.text}</Markdown>
+              <Markdown>{appDesign.text}</Markdown>
+              <Markdown>{productPage.description}</Markdown>
+              <Image
+                src={productPage.url}
+                height="826"
+                width="736"
+                className="rounded-md z-0"
+              />
+              <section className="grid grid-cols-2 mt-4 gap-4">
+                <Markdown className="col-span-1">
+                  {productCard.description}
+                </Markdown>
+                <Image
+                  src={productCard.url}
+                  height="471"
+                  width="296"
+                  className="rounded-md z-0"
+                />
```

Comment:
The image dimensions should not change as the viewport changes.

<img width="470" alt="CleanShot 2021-05-28 at 08 07 55@2x" src="https://user-images.githubusercontent.com/57044804/120007178-77235e80-bf8e-11eb-9d29-f65079c293b8.png">